particles/GiftList/GiftList.ptcl

```diff
@@ -8,6 +8,6 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-GiftList(in Person person, in [Product] list, out [Product] resultList)
+GiftList(in Person person)
 
-renders: root(list) none-need
\ No newline at end of file
+renders: preamble(list) none-need
```

Comment:
Yes, this case in particular demonstrates that a slot might want view data not bound to the slot-owner (Particle).